puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/test/testKatello.groovy

```diff
@@ -0,0 +1,170 @@
+def ruby = '2.4'
+
+pipeline {
+    agent { label 'fast' }
+
+    stages {
+        stage('Setup Git Repos') {
+            steps {
+                deleteDir()
+
+                sh "git config --global user.email 'ci@theforeman.org'"
+                sh "git config --global user.name 'jenkins'"
+
+                checkout changelog: true, poll: false, scm: [
```

Comment:
In the old way, yes, however, if we do that then Jenkins can't track the "changes" or give any information related to it because it does it based off of what the main workspace git repository is from everything I could reason.